src/edu/wright/cs/raiderplanner/controller/ChatController.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (C) 2018
```

Comment:
If you are the primary author of this file, then your name should be here. Please have a look at the other files in the project for examples.

Reply:
It was there, it was just on the next line. This will be corrected.